docs/en/docs/encoders.md

```diff
@@ -0,0 +1,202 @@
+# Encoders
+
+Encoders are basically converters between datastructures and json. Its main use is in response were different return values are automatically
+parsed into json objects.
+
+For understanding what they are doing, it is helpful to see the manual usage
+
+## Manual usage
+
+Encoders provide two public methods:
+
+- json_encode
+- apply_structure
+
+### json_encode
+
+`json_encode` is basically an enhanced json.dumps. It provides via the encoders converters which parse additional
```

Comment:
`json_encode` is basically an enhanced `json.dumps`. It provides via the encoder converters which parse additional